apps/antalmanac/src/components/Header/LoadSaveFunctionality.tsx

```diff
@@ -6,6 +6,7 @@ import {
     DialogContentText,
     DialogTitle,
     TextField,
+    CircularProgress,
 } from '@material-ui/core';
 import Checkbox from '@material-ui/core/Checkbox';
 import FormControlLabel from '@material-ui/core/FormControlLabel';
```

Comment:
Could you move the last two imports from `@material-ui/core` to the destructured import statement? You didn't change any code there, but it's just weird that they're not already included. i.e.

```tsx
import {
  // ...previous imports
  Checkbox,
  FormControlLabel
} from '@material-ui/core'
```